R/download.R

```diff
@@ -38,11 +39,17 @@ download_hydat <- function(dl_hydat_here = NULL) {
     }
   }
 
-  ans <- ask(paste("Downloading HYDAT will take ~10 minutes.","This will remove any older versions of HYDAT",
-                   "Is that okay?", sep = "\n"))
+  if (!is.logical(ask)) stop("Parameter ask must be a boolean")
```

Comment:
I wonder if this might clearer

```suggestion
  if (!is.logical(ask)) stop("Parameter ask must be a logical")
```